apps/condo/domains/billing/utils/countrySpecificQRCodeParsers/ru.qrCodeParser.js

```diff
@@ -0,0 +1,39 @@
+const iconv = require('iconv-lite')
+const { get } = require('lodash')
+
+/**
+ * @typedef {Object} TRUQRCodeFields
+ * @property {string} BIC
+ * @property {string} PayerAddress
+ * @property {string} PaymPeriod
+ * @property {string} Sum
+ * @property {string} PersAcc The resident's account
+ * @property {string} PayeeINN
+ * @property {string} PersonalAcc The bank account of the receiver
+ */
+
+/**
+ * @param {string} qrStr The QR code string got from the picture
+ * @return {TRUQRCodeFields}
+ */
+function parseRUReceiptQRCode (qrStr) {
```

Comment:
It’s cool to provide a real world test case with this function. Probably inside the tests is enough